sherpa/utils/tests/test_utils_unit.py

```diff
@@ -21,16 +21,38 @@
 
 from sherpa.utils import _utils, is_binary_file, pad_bounding_box
 from sherpa.utils.testing import requires_data
-from numpy.testing import assert_almost_equal, assert_array_equal
+from numpy.testing import assert_almost_equal, assert_array_equal, \
+    assert_array_almost_equal
 
 
 def test_calc_ftest():
     """
     this test was created using sherpa 4.8.1 (2507414) as an oracle for the python 3 migration
```

Comment:
This comment should be removed, since the test value (oracle) wasn't calculated using sherpa 4.8.1, I would guess.